base/applications/rapps/appview.cpp

```diff
@@ -1215,23 +1312,40 @@ CAppsListView::GetFocusedItemData()
 BOOL
 CAppsListView::SetDisplayAppType(APPLICATION_VIEW_TYPE AppType)
 {
+    ++g_AsyncIconTaskId; // Stop loading icons that are now invalid
     if (!DeleteAllItems())
         return FALSE;
-    ApplicationViewType = AppType;
 
+    ApplicationViewType = AppType;
     bIsAscending = TRUE;
-
     ItemCount = 0;
     CheckedItemCount = 0;
 
+    ListView_Scroll(m_hWnd, 0, 0x7fff * -1); // Fixes a bug in Wine ComCtl32 where VScroll is not reset after deleting items
```

Comment:
```suggestion
    ListView_Scroll(m_hWnd, 0, 0x7fff * -1); // FIXME: a bug in Wine ComCtl32 where VScroll is not reset after deleting items
```

Reply:
Strictly speaking, the bug is not here but in the ListView which is why I commented it this way but I get what you are saying.